subsys/bluetooth/audio/Kconfig.mctl

```diff
@@ -0,0 +1,66 @@
+# Bluetooth Audio - Media control configuration options
+
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# TODO: Decide, and add, top-level namespace prefix (currently none) to symbols.
+# Dependent upon where the module will be placed.
+
+config MCTL
+	bool "Support for media player control"
+	select EXPERIMENTAL
+	help
+	  Enables support for control of local and remote media players
+	  To enable support for control of a local media player, support for
+	  local media player must be enabled
+
+if MCTL
+
+config MCTL_LOCAL_PLAYER_CONTROL
+	bool "Support for control of local media player"
+	# TODO: Move MPL out of BT susbystem
+	depends on BT_MPL
```

Comment:
Actually, it does not have to - there does not have to be a local player.
So probably MPL should depend on MCTL instead.

Reply:
I was thinking of MCTL in general.  This is for the "LOCAL_PLAYER_CONTROL", and that does indeed depend upon the MPL.  There can be no control of a local player if there is no local player.
But it is a "depends" and not a "select", as per the previous discussion.

There should also be a dependency the other way - MPL is of no use if there is no interface for it.

Reply:
And one more thing: If we move this to a sample application, it will probably be moved along with the media_proxy, to the same subsystem.  So at least this dependency will be within that subsystem.

Reply:
> And one more thing: If we move this to a sample application, it will probably be moved along with the media_proxy, to the same subsystem. So at least this dependency will be within that subsystem.

A subsystem feature (e.g. MCTL) cannot depend on a sample application (e.g. the MPL). Since the MPL is not in the stack, the MCTL should not have a dependency towards it. 

It could be argued that the BT subsystem does not work if there's no application that calls bt_enable, but that doesn't mean that it cannot be enabled. I think it's basically the same case here.

Reply:
I would say the case is more like the example you mentioned, where Bluetooth settings depends upon SETTINGS.
There is a difference between "not being used" and "not being able to run".

Reply:
Oh, well, that was not precise.  It will run, it just won't be able to do anything.

Reply:
I see your point.  It will be able to build, and it is not strictly depending on the MPL - if there is no MPL, no player will be registered, and the application will just get error codes in return.